client/src/components/Board/Board.tsx

```diff
@@ -109,26 +126,115 @@ const Board: React.FunctionComponent = () => {
     }
   };
 
+  const addTask = (columnId: string) => {
+    const taskId = 'task-' + (Object.keys(state.mockData.tasks).length + 1);
+    const mockData = state.mockData;
+    mockData.tasks = {
+      ...mockData.tasks,
+      [taskId]: { name: 'Add title...', date: '', color: '#ffffff', id: taskId, isNew: true },
```

Comment:
What is the plan with isNew?

Reply:
Its used to check if a task should be editable & highlighted.